### PR TITLE
Document kubeadmconfigpatches.

### PR DESCRIPTION
**Pull request by amwat**
This trips up a lot of users.

/cc @BenTheElder

### CODE REVIEW

**Comment by amwat** [comment]
https://deploy-preview-1644--k8s-kind.netlify.app/docs/user/configuration/#kubeadm-config-patches

**Comment by amwat** [comment]
ref: https://github.com/kubernetes-sigs/kind/issues/1361

File: site/content/docs/user/configuration.md [hunk 1]
```diff
@@ -218,3 +218,54 @@ You may also want to see the [Ingress Guide].
...
 [Ingress Guide]: ./../ingress
+
+### Kubeadm Config Patches
+
+KIND uses [`kubeadm`](./../../design/principles/#leverage-existing-tooling) 
+to configure cluster nodes.
+Formally  KIND runs `kubeadm init` on the first control-plane node 
+which can be customized by using the kubeadm
+[InitConfiguration](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file) 
+([spec](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1#InitConfiguration))
```
Comment (neolit123) [inline site/content/docs/user/configuration.md:229]: there is a good chance that kubeadm v1beta1 will be removed in 1.20, so it's is better to recommend v1beta2 at this point.

File: site/content/docs/user/configuration.md [hunk 2]
```diff
@@ -218,3 +218,54 @@ You may also want to see the [Ingress Guide].
...
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "my-label=true"
+{{< /codeFromInline >}}
+
+On every additional node configured in the KIND cluster, 
+worker or control-plane (in HA mode),
+KIND runs `kubeadm join` which can be configured using the 
+[JoinConfiguration](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-join/#config-file)
+([spec](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1#JoinConfiguration))
```
Comment (neolit123) [inline site/content/docs/user/configuration.md:248]: [1]

**Comment by BenTheElder** [comment]
/lgtm
/approve
thank you!
sorry for the PR delay, we've been fighting CI fires.

**Pull request merged**